source/usage-examples/findOne.rst

```diff
@@ -0,0 +1,23 @@
+===============
+Find a Document
+===============
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+You can find a single document using the ``collection.findOne()``
+method. ``findOne`` takes a query document, an optional sort order,
+and an optional projection. ``findOne`` returns a Promise that resolves
```

Comment:
"findOne returns a Promise" 

should either be 

"findOne returns a promise" (non capitalized P)

unless you either external link to mdn docs on promises or is bolded